package.json

```diff
@@ -76,6 +76,7 @@
     "mongodb-extended-json": "^1.9.0",
     "mongoose": "^4.7.5",
     "morgan": "^1.8.1",
+    "nodemon": "^1.11.0",
```

Comment:
Actually, is it possible to install nodemon globally on linux? I'm guessing it should be. That's more ideal than having it in the HAH project! 😄